src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -147,12 +136,10 @@ export function PortfolioLogo({
   }
 
   const L2Logo =
-    chainId === SupportedChainId.MAINNET ? null : (
-      <div>
-        {chainLogo && <ChainLogoSquareBackground />}
-        {!squareLogoUrl && logoUrl && <SquareBackgroundForNonSquareLogo />}
-        {chainLogo && <StyledChainLogo isSquare={!!squareLogoUrl} src={chainLogo} alt="chainLogo" />}
-      </div>
+    chainId === SupportedChainId.MAINNET || !chainLogo ? null : (
```

Comment:
i'm finding this logic a bit hard to parse visually. usually i'll think "show the logo if the conditions for the logo to be shown are met, otherwise it's null". so that would look something like "show it if we're not on mainnet and we have a chainLogo"